app/helpers/applicants_helper.rb

```diff
@@ -50,22 +51,24 @@ def background_class_for_context_status(context, number_of_days_before_action_re
       "bg-danger border-danger"
     elsif context.time_to_accept_invitation_exceeded?(number_of_days_before_action_required)
       "bg-warning border-warning"
-    elsif context.rdv_seen?
+    elsif context.rdv_seen? || context.closed?
       "bg-success border-success"
     else
       ""
     end
   end
 
   def background_class_for_participation_status(participation)
-    if participation.seen?
+    if participation.rdv_context.closed? || !(
+      participation.seen? || participation.cancelled? || participation.needs_status_update?
+    )
+      ""
```

Comment:
je pense que ce serait plus lisible d'avoir en début de méthode
```ruby 
return "" if participation.rdv_context.closed?
```
et garder le reste comme c'était parce que là c'est difficilement lisible 😄

Reply:
oui je pense aussi, mais rubocop doesn't like it :)